The GAMESS-US build process requires that the MAXCPUS and MAXNODES values be
set before compiling the DDI library. MAXNODES should be the number of nodes
in your cluster, while MAXCPUS should be the number of CPU cores in the node
with the most cores.
--- gamess.orig/ddi/compddi	2013-05-04 08:14:15.000000000 +1200
+++ gamess/ddi/compddi	2015-01-23 15:37:42.510554228 +1300
@@ -82,8 +82,8 @@
 #     It is OK if these values are larger than your system,
 #     but a serious error if they are smaller!
 #
-set MAXCPUS  = 16
-set MAXNODES = 256
+set MAXCPUS  = 40
+set MAXNODES = 392
 #
 if ($TARGET == cray-xt) then
    set MAXCPUS=12
